test/common/upstream/cluster_manager_impl_test.cc

```diff
@@ -2824,6 +2827,52 @@ TEST_F(ClusterManagerInitHelperTest, UpdateAlreadyInitialized) {
   cluster2.initialize_callback_();
 }
 
+TEST_F(ClusterManagerInitHelperTest, InitSecondaryWithoutEdsPaused) {
```

Comment:
Can you add single line `//` comments above each of the new tests to describe the intuition of what they are validating?